app/views/layouts/_error_messages.html.erb

```diff
@@ -0,0 +1,17 @@
+<% if model.errors.messages.length > 0 %>
```

Comment:
It's awesome that you have this generic error messages template! However, I think this does not quite count as a "layout" (a view file which is intended to contain the page-specific view files being rendered), it is closer to a partial (a view file that can be embedded within other page-specific views).

In that case I believe the Rails convention is to place such partials in the `app/views/shared` directory (you may need to create this directory).